_redirects

```diff
@@ -8,3 +8,10 @@
 /videos/tags /videos 302!
 /weekend-prompts /weekendfollowup 302!
 /*  /404/index.html 404
+/atriumevents/?site=oakley https://api.crossroads.net/atriumevents/eventlisting/oakley
```

Comment:
These lines need to come before the catch all `*` rule ☝🏻 otherwise, they will never get processed.